_posts/2019-09-04-pulp-3-performance-blog.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Pulp 3 Performance Blog
+author: Imaanpreet Kaur
+team: Red Hat Performance Team
+members: Pradeep Surisetty, Jan Hutař, Imaanpreet Kaur
+---
+
+## Motivation
+
+We want to investigate the performance of various Pulp3 operations including: Sync, Publish, Download, Content Listing, and Copy operations.Pulp team came up with a test plan and Red Hat Performance team tested these test cases. So, let’s discuss about the pulp3 performance in brief.
```

Comment:
s/operations.Pulp/operations. Pulp

Reply:
Sorry, I couldn't get you. What do I have to change here?

Reply:
Missing space between the two sentences after the period